x-pack/plugins/maps/public/layers/util/can_skip_fetch.test.js

```diff
@@ -8,101 +8,74 @@ import { canSkipSourceUpdate, updateDueToExtent } from './can_skip_fetch';
 import { DataRequest } from './data_request';
 
 describe('updateDueToExtent', () => {
-  it('should be false when the source is not extent aware', async () => {
-    const sourceMock = {
-      isFilterByMapBounds: () => {
-        return false;
-      },
+  it('should be false when buffers are the same', async () => {
```

Comment:
Diff is really noisy for this change. Just removed `source` from updateDueToExtent signature